src/assets/icons/enturLogo.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import EnturWhite from './../logos/Enturlogo_white.svg'
+import EnturBlack from './../logos/Enturlogo_black.svg'
+
+function EnturLogo({ className, style, height }: Props): JSX.Element {
+    let enturLogo = ''
```

Comment:
Kan fint korte det litt ned her:
`const enturLogo = style === 'black' ? EnturBlack : EnturWhite`
Ettersom man kun bruker `EnturBlack` dersom `style === black` og man defaulter til `EnturWhite` :)

Reply:
Ser også at du bruker `'dark' | 'light' | 'positive' | 'negative'` begreper i `TavlaLogo`. Kanskje du kan bruke `dark`og `light`her. Det hadde også vært fint å bruke en constant også i stedet for hardkode en string, som kan bli brukt i både `TavlaLogo` og i denne filen.

Reply:
stylingen blir vel endret når vi får flettet sammen med thems, uansett regner æ med, så da blir det vel litt annerledes uansett